engine/src/main/java/org/hibernate/search/v6poc/search/dsl/predicate/SearchPredicateContainerContextExtension.java

```diff
@@ -27,6 +33,8 @@
 	/**
 	 * Attempt to extend a given context, throwing an exception in case of failure.
 	 *
+	 * <strong>WARNING:</strong> this method is not API, see comments at the type level.
```

Comment:
I think a `<p>` before this sentence would be in order, WDYT?

Same for the others below.

Reply:
Right. Done.